### PR TITLE
vault ci

### PR DESCRIPTION
**Pull request by lioramilbaum**

### CODE REVIEW

**Comment by lioramilbaum** [comment]
@ShimiTaNaka I am aware CI is failing. It won't as soon as all the PRs will be submitted/approved/merged. Please review.

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -2,10 +2,32 @@
...
-pip3 install -r $BASE_FOLDER/requirements.txt
+# shellcheck source=/vagrant/.env
+source $HOME_DIR/.env
+if [[ -f $HOME_DIR/.env.local ]]; then
+# shellcheck source=/vagrant/.env.local
+  source $HOME_DIR/.env.local
+fi
+
+pip3 install --upgrade pip
+pip install -r $HOME_DIR/requirements.txt
```
Comment (sbardugo) [inline scripts/bootstrap.sh]: Why did you change pip3 to pip?

Comment (lioramilbaum) [reply scripts/bootstrap.sh]: As soon as pip3 upgrades, it sets pip to be pip3.

File: tests/vault.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        driver.find_element_by_css_selector(
+            "button#auth-submit.button.is-primary"
+        ).click()
+        element = driver.find_element_by_css_selector(
+            'h1.title.is-3'
+        )
+        assert element.text == "Secrets Engines"
+
+    print("Tests Passed Successfully")
+except NoSuchElementException as error:
```
Comment (sbardugo) [inline tests/vault.py]: Why did you use Except? if we will get the exception, isnt that means that the tests failed?
If we will inhert from unittest.TestCase we can used there assartions, and check for exception, if we want it to accoure

Comment (lioramilbaum) [reply tests/vault.py]: I never used unitest.TestCase till now. But, it sounds reasonable. Let me do some learning on the topic.
If you know how to modify the code than be my guest and peach in.

File: tests/vault.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```
[the remark this answers is not in the document]
Comment (lioramilbaum) [reply tests/vault.py]: Prefix is like a header (comment) for documentation purposes?

File: tests/vault.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+import logging
+from dotenv import load_dotenv
+from pathlib import Path
+import os
+import sys
+import time
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from selenium.common.exceptions import NoSuchElementException
```
[the remark this answers is not in the document]
Comment (lioramilbaum) [reply tests/vault.py]: +1

[the remark this answers is not in the document]
Comment (lioramilbaum) [reply tests/vault.py]: I have found a utility to help with it 
https://pypi.org/project/importanize/

File: tests/vault.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (sbardugo) [reply tests/vault.py]: No, just add test_ file <name>

File: scripts/bootstrap.sh (hunk 1 above)
Comment (lioramilbaum) [reply scripts/bootstrap.sh]: +1

Comment (lioramilbaum) [reply scripts/bootstrap.sh]: removed line 18 because it caused issues of mixing system installer (apt-get) and pip

File: tests/vault.py (hunk 2 above)
Comment (lioramilbaum) [reply tests/vault.py]: I didn't intend to test the exception. Therefor, removed it from the test script.

File: tests/vault.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (lioramilbaum) [reply tests/vault.py]: +1

File: tests/vault.py (hunk 2 above)
Comment (sbardugo) [reply tests/vault.py]: Why did you use except twice for the same exception ?

Comment (lioramilbaum) [reply tests/vault.py]: probably oversight. It is not there any more.
